GSoC-Participants.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: GSoC participants
```

Comment:
The commit subject is using "GSoC-particpants:" instead of GSoC-participants:" (missing "i").